### ISSUE TITLE
Incorrect JSON output for multi-dimensional arrays of enumerated types

### ISSUE THREAD

**Issue by Dekker1** [MiniZinc/libminizinc#430]
The JSON output for multi-dimensional arrays of enumerated types seems incorrect (and at least inconsistent):

```
enum DAY = {A,B,C,D};
array[1..2,1..2] of var DAY: res;
array[1..2,1..2] of var int: res2;
```
Solving this model using `minizinc --ouput-mode json` gives:

```
{
  "res" : [{"e":"A"}, {"e":"A"}, {"e":"A"}, {"e":"A"}],
  "res2" : [[-2147483646, -2147483646], [-2147483646, -2147483646]]
}
----------
```

It is expected that `res` is sub-divided into its dimensions the same as `res2`

**Comment by Dekker1** [comment]
Solved in 666806eff55c59f56dd20c4b8dfbf3ff34a6ae63

**Comment by cyderize** [comment]
This should now be fixed in [2.5.3](https://github.com/MiniZinc/MiniZincIDE/releases/tag/2.5.3).
